### ISSUE TITLE
Removed JavaMail 1.6.4 tag, replaced with JakartaMail 1.6.4 tag, useless new 1.6.4 source zip/tarball

### ISSUE THREAD

**Issue by wltjr** [eclipse-ee4j/mail#395]
**Describe the bug**
I am not sure why tag 1.6.4 for JavaMail was removed and replaced with 1.6.4 tag for JakartaMail. I understand the renaming, but removing tags is bad. Just create a new tag and do no delete the old ones. Especially tags that were created months ago, it breaks others usage of those tags--[this package is now broken](https://github.com/Obsidian-StudiosInc/os-xtoo/tree/70cbf8d40a8128de979931f0aa346981f3d479ba/dev-java/javamail). Even more important, the new tag 1.6.4's zip and tar.gz download for 1.6.4 do not contain sources at all. The contents are very strange.

Source zip and tarball should contain sources, not what ever is in the 1.6.4 sources.

**To Reproduce**
1. Download 1.6.4 zip or tar.gz file from sources of 1.6.4 release.
2. Unpack
3. Realize they do not contain source code or anything along those lines

**Expected behavior**
The sources actually contain sources as they previous did, see 1.6.3 sources for reference. With tags that remain and are NOT deleted months later, like 1.6.4 was deleted and replaced with the exact same version and tag, but with difference sources.

**Comment by bshannon** [comment]
Partly this is due to git being a horrible source code management system,
partly due to GitHub putting the source code and web site in separate branches,
and partly this is me not knowing how to use git and GitHub properly.  :-)

There was never a JavaMail 1.6.4 release.

In the process of producing the Jakarta Mail 1.6.4 release, using the new
Jakarta EE release process, we made some mistakes and learned some things.
This caused us to revise what we thought was the final 1.6.4 release as we went
through the release process.  I probably should not have created the 1.6.4 tag
until the very end, when we were sure it was final.

I see that the 1.6.4 tag is attached to the gh-pages branch.  I have no idea how
that happened.  I suppose that means I need to delete and recreate the 1.6.4
tag, and perhaps the 1.6.4 release itself.  If you know how to fix this, let me know.

Sorry for the confusion...
